test/tests/pbs_qstat_performance.py

```diff
@@ -0,0 +1,146 @@
+# Public License agreement ("AGPL"), except where a separate commercial license
+# agreement for PBS Pro version 14 or later has been executed in writing with Altair.
+#
+# Altair’s dual-license business model allows companies, individuals, and
+# organizations to create proprietary derivative works of PBS Pro and distribute
+# them - whether embedded or bundled with other software - under a commercial
+# license agreement.
+#
+# Use of Altair’s trademarks, including but not limited to "PBS™",
+# "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's
+# trademark licensing policies.
+
+import os
+import string
+import subprocess
+
+from ptl.utils.pbs_testsuite import *
+
+class TestQstatPerformance(PBSTestSuite):
+
+    """
+    Testing Qstat Performance
+    """
+    elapsed_time_dictionary = (
+                ' ` qselect`',
+                ' workq `qselect`',
+                ' `qselect` workq',
+                ' -s `qselect`',
+                ' -f `qselect`',
+                ' -f workq ` qselect`'
+                )
+    time_command = ''
+    sleep_command = ''
+    sleep_arguments = '1000'
+
+    def which(self, command):
+        "Gives absolute path of a command."
+        for path in os.getenv("PATH").split(os.path.pathsep):
+            absolute_path = path + os.sep + command
+            if os.path.exists(absolute_path):
+                return absolute_path
+        return None
+
+    def setUp(self):
+        self.time_command = self.which("time")
+        self.sleep_command = self.which("sleep")
+        if self.time_command == None:
+            self.fail("Time command not found")
+        if self.sleep_command == None:
+            self.fail("Sleep command not found")
+
+    def compute_elapse_time(self, query):
+        " Computes qstat time in secs"
+        command = self.time_command
+        command += " -f \"%e\" "
+        command += os.path.join(
+                                self.server.client_conf['PBS_EXEC'],
+                                'bin',
+                                'qstat')
+        command += query
+
+        # compute elapse time without -E option
+        without_E_option = self.du.run_cmd(self.server.hostname,
+                              command,
+                              as_script=True,
+                              logerr=False)
+        if without_E_option['rc'] != 0:
+            raise RuntimeError("Qstat failed")
+        # compute elapse time with -E option
+        command += " -E"
+        with_E_option = self.du.run_cmd(self.server.hostname,
+                              command,
+                              as_script=True,
+                              logerr=False)
+
+        if with_E_option['rc'] != 0:
+            raise RuntimeError("Qstat failed")
+        self.logger.info("Without E option :" + without_E_option['err'][0])
+        self.logger.info("With E option    :" + with_E_option['err'][0])
+        self.assertGreaterEqual(
```

Comment:
assertGreaterEqual is introduced in Python 2.7 but PBS support Python 2.6 onward so please use assertTrue instead.